benchmarks/Benchmarks.Trace/SpanBenchmark.cs

```diff
@@ -0,0 +1,43 @@
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Jobs;
+using Datadog.Trace;
+using Datadog.Trace.BenchmarkDotNet;
+using Datadog.Trace.ExtensionMethods;
+
+namespace Benchmarks.Trace
+{
+    /// <summary>
+    /// Span benchmarks
+    /// </summary>
+    [DatadogExporter]
+    [MemoryDiagnoser]
+    [SimpleJob(RuntimeMoniker.Net472)]
+    [SimpleJob(RuntimeMoniker.NetCoreApp31)]
+    public class SpanBenchmark
+    {
+        private static Tracer _tracer = Tracer.Instance;
```

Comment:
We should also look into disabling all background activity for the Tracer instance being benchmarked. I've seen in my tests that it produces some noise

Reply:
There is a new `DummyAgentWriter` in (https://github.com/DataDog/dd-trace-dotnet/pull/817/commits/a417528e343732261614e967417cb137479463bd) to reduce this background activity.